test-lib-openshift.sh

```diff
@@ -1060,4 +1060,30 @@ function ct_os_check_cmd_internal() {
   return 1
 }
 
+# ct_os_test_image_stream
+# ------------------------
+# Creates an image stream and deploys a specified template. Then checks that a pod runs.
+# Argument: image_stream_file - local file name with an image stream
+# Argument: template_file - local file name with a template
+# Argument: service_name - how the pod will be named (prefix)
+# Argument: template_params (optional) - parameters for the template, like image stream version
+function ct_os_test_image_stream() {
+  local image_stream_file=$1
+  local template_file=$2
+  local service_name=$3
+  local template_params=${4:-}
+
+  echo "Running image stream test for stream $image_stream_file and template $template_file"
```

Comment:
Please add there a check that files really exist. I would like to avoid the typos.

Reply:
and of course, `image_stream_file`, `template_file`, and `service_name` should not be empty, otherwise `oc create -f` will wait and maybe block the test forever.

Reply:
Ok, thanks for the suggestions, will add some more sanity checks.

Reply:
Actually, for the file check itself, I believe the extra check is not necessary, because if the file does not exist, user will see pretty clear error message from the `oc process` command. I'll only add checks for having the values specified.